helm-charts/falcon-sensor/templates/clusterrolebinding.yaml

```diff
@@ -17,7 +17,7 @@ subjects:
   name: system:authenticated
 {{- end }}
 - kind: ServiceAccount
-  name: crowdstrike-falcon-sa
+  name: {{include "falcon-sensor.serviceAccountName" .}}
```

Comment:
This should be `{{ .Values.serviceAccount }}`

Reply:
done